examples/md-flexible/src/configuration/objects/Object.h

```diff
@@ -39,24 +39,27 @@ class Object {
 
   /**
    * Generate the object in the given AutoPas container.
-   * @note Particle IDs for the new particles will start at the current value of autopas.getNumberOfParticles().
-   *
-   * @param autopas
+   * @param particles The container to which the new particles will be appended to.
    */
-  virtual void generate(autopas::AutoPas<ParticleType> &autopas) const = 0;
+  virtual void generate(std::vector<ParticleType> &particles) const = 0;
 
   /**
    * Create a particle that acts as blueprint for all particles to be created for the object.
-   * @param autopas
-   * @return
-   */
-  [[nodiscard]] ParticleType getDummyParticle(const autopas::AutoPas<ParticleType> &autopas) const {
-    ParticleType dummyParticle;
-    dummyParticle.setV(_velocity);
-    dummyParticle.setID(autopas.getNumberOfParticles());
-    dummyParticle.setTypeId(_typeId);
-    return dummyParticle;
+   * @param particleId: Defines the id of the generated dummy particle.
+   * @return a particle initialized with default values.
+   */
+  [[nodiscard]] ParticleType getDummyParticle(const size_t &particleId) const {
+    ParticleType particle{};
+    particle.setID(particleId);
+    particle.setTypeId(_typeId);
+    particle.setOwnershipState(autopas::OwnershipState::owned);
+    particle.setV(_velocity);
+    particle.setF({0.0, 0.0, 0.0});
+    particle.setOldF({0.0, 0.0, 0.0});
+
+    return particle;
```

Comment:
```suggestion
  [[nodiscard]] ParticleType getDummyParticle(const size_t &particleId) const {
    ParticleType particle{};
    particle.setID(particleId);
    particle.setTypeId(_typeId);
    particle.setV(_velocity);
    return particle;
```

Why repeat what the constructor does anyway?